terminus-ui/file-upload/src/file-upload.component.ts

```diff
@@ -787,11 +791,22 @@ export class TsFileUploadComponent extends TsReactiveFormBaseComponent implement
 
   private parseRatioToString(ratios: Array<ImageRatio> | undefined): Array<string> | undefined {
     if (!ratios) {
-      return;
+      return undefined;
     }
     const parsedRatio: Array<string> = [];
-    ratios.map((r) => parsedRatio.push(r.widthRatio.toString() + ':' + r.heightRatio.toString()));
+    ratios.map(r => parsedRatio.push(`${r.widthRatio.toString()  }:${  r.heightRatio.toString()}`));
     return parsedRatio;
   }
 
+
+  /**
+   * Function for tracking for-loops changes
+   *
+   * @param index - The item index
+   * @return The unique ID
+   */
+  public trackByFn(index): number {
```

Comment:
Is there a test for this?

Reply:
It's being tested implicitly by other tests